src/app/branded/step-1/branded-checkout-step-1.component.js

```diff
@@ -154,12 +159,113 @@ class BrandedCheckoutStep1Controller {
   checkSuccessfulSubmission () {
     if (every(this.submission, 'completed')) {
       if (every(this.submission, { error: false })) {
-        this.next()
+        if (this.useV3 === 'true') {
+          this.submitOrderInternal()
+        } else {
+          this.next()
+        }
       } else {
         this.submitted = false
       }
     }
   }
+
+  loadCart () {
+    this.errorLoadingCart = false
+
+    const cart = this.cartService.get()
+    cart.subscribe(
+      data => {
+        // Setting cart data and analytics
+        this.cartData = data
+        this.brandedAnalyticsFactory.saveCoverFees(this.orderService.retrieveCoverFeeDecision())
+        this.brandedAnalyticsFactory.saveItem(this.cartData.items[0])
```

Comment:
This fails when `loadCart` is called from `handleRecaptchaFailure` because `data.items` is empty. It would probably be best to check whether `items` has at least one item before doing this step.

Reply:
@wrandall22 is it worth fixing this or are we going to remove `handleRecaptchaFailure()`?

Reply:
It is probably worth it for the sake of covering an edge case.